src/DssDeploy.t.sol

```diff
@@ -544,12 +552,24 @@ contract DssDeployTest is DssDeployTestBase {
         assertEq(gov.balanceOf(address(user1)), 1 ether);
     }
 
-    function testFireESM() public {
+    function testFireESMBug() public {
+        deploy();
+        gov.mint(address(user1), 10);
+
+        user1.doESMJoin(address(gov), address(esmBug), 10);
+        assertEq(vat.wards(address(pause.proxy())), 1);
+        esmBug.fire();
+        assertEq(vat.wards(address(pause.proxy())), 1);
```

Comment:
We might want to add a `assertEq(end.live(), 0);`

Reply:
Done